src/user/announcements.rst

```diff
@@ -13,7 +13,7 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 
 We have made the ``conda-forge`` Google Group read-only. Please use the new 
 ``conda-forge`` `discourse forum <https://conda.discourse.group/c/pkg-building/conda-forge/25>`_, 
-our `Gitter room <https://gitter.im/conda-forge/conda-forge.github.io>`_, or it's `Matrix/Element 
+our `Element Chatroom <https://app.element.io/#/room/#conda-forge:matrix.org>`_, or it's `Matrix/Element 
```

Comment:
```suggestion
our `Element chatroom <https://app.element.io/#/room/#conda-forge:matrix.org>`_, or it's `Matrix/Element 
```

Now we have two mentions to the same thing in the same sentence. 

But, once again, this is an announcement, so we shouldn't touch it.

Revert please. Thanks!

Reply:
And the same applies to the rest of the file.